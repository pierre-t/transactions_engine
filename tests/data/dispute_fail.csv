type,client,tx,amount
deposit,1,1,1.0
deposit,1,2,2.0
dispute,1,1,
dispute,1,1,
dispute,1,1,
dispute,1,1,
dispute,1,1,
dispute,1,2,
resolve,1,1,
dispute,1,2,
resolve,1,2,
deposit,2,10,100.0
withdrawal,2,11,1.0
dispute,2,11,
deposit,2,12,1.0
dispute,1,10,
dispute,1,99,
dispute,3,10,