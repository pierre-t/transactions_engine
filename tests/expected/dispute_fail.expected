client,available,held,total,locked
1,3,0,3,false
2,100,0,100,false